connector/mqtt/vernemq/k2v-bridge/app/config.js

```diff
@@ -1,32 +1,73 @@
-const hostname = process.env.HOSTNAME || 'k2v-bridge';
-const parseSecureMode = (mode) => ((mode || false) && (mode.toString().toLowerCase().trim() === 'true' || Number(mode) > 0));
+const { toBoolean } = require('./utils');
 
 const app = {
-  logLevel: process.env.K2V_LOG_LEVEL || 'info',
-  baseDir: process.env.BASE_DIR || '/opt/k2v-bridge',
-  hostname,
+  basedir: process.env.K2V_APP_BASEDIR,
+  'connection.retry.count': parseInt(process.env.K2V_APP_CONNECTION_RETRY_COUNT, 10),
+  'connection.retry.timeout': parseInt(process.env.K2V_APP_CONNECTION_RETRY_TIMEOUT, 10),
+  'ejbca.address': process.env.K2V_APP_EJBCA_ADDRESS,
+  hostname: process.env.K2V_APP_HOSTNAME,
+};
+
+const logger = {
+  'transports.console.level': process.env.K2V_LOGGER_TRANSPORTS_CONSOLE_LEVEL,
+  verbose: (toBoolean(process.env.K2V_LOGGER_VERBOSE)),
+};
+
+const messenger = {
+  'consume.topic.suffix': process.env.K2V_MESSENGER_CONSUME_TOPIC_SUFFIX,
 };
 
 const mqtt = {
-  clientUsername: process.env.K2V_MQTT_USERNAME || 'k2v-bridge',
-  clientId: process.env.K2V_MQTT_CLIENT_ID || hostname,
-  host: process.env.K2V_MQTT_HOST || 'vernemq-k8s',
-  port: parseInt(process.env.K2V_MQTT_PORT, 10) || 8883,
-  keepAlive: parseInt(process.env.K2V_MQTT_KEEPALIVE, 10) || 60,
-  secure: parseSecureMode(process.env.K2V_MQTT_SECURE || true),
-  publishTopicSuffix: process.env.K2V_MQTT_PUBLISH_TOPIC_SUFFIX || '/config',
-  publishQos: parseInt(process.env.K2V_MQTT_PUBLISH_QOS, 10) || 1,
-  tls: {
-    ca: {
-      location: process.env.K2V_MQTT_CA_FILE || `${app.baseDir}/app/cert/ca.crt`,
-    },
-    certificate: {
-      location: process.env.K2V_MQTT_CERT_FILE || `${app.baseDir}/app/cert/${hostname}.crt`,
-    },
-    privateKey: {
-      location: process.env.K2V_MQTT_KEY_FILE || `${app.baseDir}/app/cert/${hostname}.key`,
-    },
-  },
+  // Client
+  'client.id': process.env.K2V_MQTT_CLIENT_ID,
+  'client.keepalive': parseInt(process.env.K2V_MQTT_CLIENT_KEEPALIVE, 10),
+  'client.publish.qos': parseInt(process.env.K2V_MQTT_CLIENT_PUBLISH_QOS, 10),
+  'client.publish.topic.suffix': process.env.K2V_MQTT_CLIENT_PUBLISH_TOPIC_SUFFIX,
+  'client.secure': toBoolean(process.env.K2V_MQTT_CLIENT_SECURE),
+  'client.username': process.env.K2V_MQTT_CLIENT_USERNAME,
+
+  // Server
+  'server.address': process.env.K2V_MQTT_SERVER_ADDRESS,
+  'server.port': parseInt(process.env.K2V_MQTT_SERVER_PORT, 10),
+
+  // TLS
+  'tls.ca.file': process.env.K2V_MQTT_TLS_CA_FILE,
+  'tls.certificate.file': process.env.K2V_MQTT_TLS_CERTIFICATE_FILE,
+  'tls.key.file': process.env.K2V_MQTT_TLS_KEY_FILE,
 };
 
-module.exports = { app, mqtt, parseSecureMode };
+const kafka = {
```

Comment:
Some of the properties doesn't apply to the Kafka Consumer. See the the comments in the README.md